docs/contents/components/forms/hue-slider/accessibility.mdx

```diff
@@ -24,19 +24,18 @@ The `HueSlider` follows the [WAI-ARIA - Slider Pattern](https://www.w3.org/WAI/A
 
 ## ARIA Roles and Attributes
 
-| Component                   | Role and Attributes | Usage                                                                                            |
-| --------------------------- | ------------------- | ------------------------------------------------------------------------------------------------ |
-| `HueSliderThumb`            | `role`              | Indicates that this is a slider.                                                                 |
-|                             | `aria-label`        | Sets `"Slider thumb"`.                                                                           |
-|                             | `aria-orientation`  | Sets `"horizontal"` or `"vertical"` based on the `orientation` value. Default is `"horizontal"`. |
-|                             | `aria-valuemin`     | Sets the `min` value. Default is `0`.                                                            |
-|                             | `aria-valuemax`     | Sets the `max` value. Default is `100`.                                                          |
-|                             | `aria-valuenow`     | Sets the current value.                                                                          |
-|                             | `aria-valuetext`    | Sets the current value, such as `18°, Red`.                                                      |
-|                             | `aria-readonly`     | Set to `"true"` if `isReadOnly` is set.                                                          |
-|                             | `aria-disabled`     | Set to `"true"` if `isDisabled` is set.                                                          |
-|                             | `aria-invalid`      | Set to `"true"` if `isInvalid` is set.                                                           |
-| `input` <Tag>Internal</Tag> | `type="hidden"`     | Excludes the element from the accessibility tree.                                                |
-|                             | `aria-readonly`     | Set to `"true"` if `isReadOnly` is set.                                                          |
-|                             | `aria-disabled`     | Set to `"true"` if `isDisabled` is set.                                                          |
-|                             | `aria-invalid`      | Set to `"true"` if `isInvalid` is set.                                                           |
+| Component                            | Role and Attributes | Usage                                             |
+| ------------------------------------ | ------------------- | ------------------------------------------------- |
+| `HueSliderThumb` <Tag>Internal</Tag> | `role="slider"`     | Indicates that this is a slider.                  |
+|                                      | `aria-label`        | Sets `"Slider thumb"`.                            |
```

Comment:
The value of `aria-label` is currently set to `"Slider thumb"`, but updating it to more specific labels like `"Alpha Slider thumb"` or `"Hue Slider thumb"` can make it clearer for users of screen readers, helping them easily identify which slider they are interacting with.

Reply:
The actual implementation is the `"Slider thumb"`.

Please discuss that by creating another issue, etc.